### PR TITLE
Fix Python lint issues for sgcollect_info, tasks.py, password_remover.py

### PR DESCRIPTION
**Pull request by bbrks**
Fixed Python lint issues from the [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/)

- Removed unused imports
- Trimmed trailing whitespace
- Separated top-level functions with 2 blank spaces
- Separated class-level functions with 1 blank space
- Changed `==` to `is`, `!=` to `is not`

### CODE REVIEW

File: tools/sgcollect_info [hunk 1]
```diff
@@ -506,22 +501,22 @@ def get_config_path_from_cmdline(cmdline_args):
...
+
     # get content and parse into json
     try:
         response = urllib2.urlopen(expvar_url(sg_url))
         data = json.load(response)
     except urllib2.URLError as e:
         print("WARNING: Unable to connect to Sync Gateway: {0}".format(e))
 
-    if data is not None and data.has_key("cmdline"):
+    if data is not None and "cmdline" in data:
```
Comment (tleyden) [inline tools/sgcollect_info:519]: Can you double check which version of python is being used and whether it supports this syntax?

File: tools/sgcollect_info [hunk 2]
```diff
@@ -777,9 +773,9 @@ def main():
             do_upload_and_exit(zip_filename, upload_url)
 
     if options.redact_level != "none":
-        print "Zipfile built: {0}".format(redact_zip_file)
+        print("Zipfile built: {0}".format(redact_zip_file))
 
-    print "Zipfile built: {0}".format(zip_filename)
+    print("Zipfile built: {0}".format(zip_filename))
```
Comment (tleyden) [inline tools/sgcollect_info:778]: Ditto on python version syntax check.

**Review by tleyden — CHANGES_REQUESTED** [review]
Just a few questions on python version supported syntax

File: tools/sgcollect_info (hunk 1 above)
Comment (bbrks) [reply tools/sgcollect_info:519]: `has_key` was removed in Python 3, `in` works on both 2.x and 3.x

File: tools/sgcollect_info (hunk 2 above)
Comment (bbrks) [reply tools/sgcollect_info:778]: It's Python 3 syntax but also backwards compatible with 2.x, so long as there's only a single argument.

**Comment by bbrks** [comment]
@tleyden Both of these syntax changes are to fix running under Python 3, but are also backwards compatible with 2.x

**Review by tleyden — APPROVED** [review]

**Pull request merged**